python/ql/test/experimental/dataflow/tainttracking/defaultAdditionalTaintStep/test_with.py

```diff
@@ -49,7 +49,7 @@ def __exit__(self, exc_type, exc, tb):
 def test_with_arg():
     ctx = Context_arg(TAINTED_STRING)
     with ctx as tainted:
-        ensure_tainted(tainted) # $ tainted
+        ensure_tainted(tainted) # $ MISSING: tainted
```

Comment:
This is actually expected, I think, until we model `__enter__`.

Reply:
agreed :+1: